File: C#/FizzBuzz.cs
using System;

class FizzBuzz
{
  static void Main()
  {
    var input = System.Console.ReadLine();
    int x = System.Int32.Parse(input);

    if (x % 15 == 0)
      Console.WriteLine("FizzBuzz");
    else if (x % 5 == 0)
      Console.WriteLine("Buzz");
    else if (x % 3 == 0)
      Console.WriteLine("Fizz");
    else
      Console.WriteLine(x);
  }
}

File: Julia/FizzBuzz.jl
x = parse.(Int, readline())
  
for i in 1:x
  if i % 15 == 0
    print("FizzBuzz")
  elseif i % 3 == 0
    print("Fizz")
  elseif i % 5 == 0
    print("Buzz")
  else
    print(i)
  end
  print("\t")
end